presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/WriteFunction.java

```diff
@@ -13,7 +13,16 @@
  */
 package io.prestosql.plugin.jdbc;
 
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+
 public interface WriteFunction
 {
     Class<?> getJavaType();
+
+    default void setNull(PreparedStatement statement, int index)
```

Comment:
I am not sure about it. Now you could reuse existing write functions when you had to implement custom write functions. Now you need to extend all write functions. That was the point of having them separated.

Same comment goes to read function. However that does hit me.

Reply:
I hear what you're saying. This makes the interface coherent & simplifies the code. I think this is a good improvement even considered this downside that you mention

Reply:
This simplifies the code on side and complicates on the other side. I would expect gains here to be  much lower than loses. ; /